xbmc/video/VideoDatabase.cpp

```diff
@@ -4602,11 +4607,31 @@ void CVideoDatabase::UpdateTables(int iVersion)
     m_pDS->exec("ALTER TABLE tvshow ADD userrating integer");
     m_pDS->exec("ALTER TABLE musicvideo ADD userrating integer");
   }
+
+  if (iVersion < 97)
+  {
+    // Add idSeason to episode table, so we don't have to join via idShow and season in the future
+    m_pDS->exec("ALTER TABLE episode ADD idSeason integer");
+
+    m_pDS->query("SELECT idSeason, idShow, season FROM seasons");
+    while (!m_pDS->eof())
+    {
+      m_pDS->exec(PrepareSQL("UPDATE episode "
```

Comment:
This needs to be `m_pDS2->exec()` because `m_pDS` is already used to loop through the results of your select query.